packages/web/src/utils/index.js

```diff
@@ -138,3 +138,15 @@ export function parseValueArray(originalArr = [], currentValue) {
 export function escapeRegExp(string) {
 	return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
 }
+
+export function parseCompAggToHits(aggFieldName, buckets = []) {
```

Comment:
Please move this function to `reactivecore`.

Reply:
https://github.com/appbaseio/reactivecore/pull/48